src/openrct2/ride/TrackData.cpp

```diff
@@ -2977,346 +2977,346 @@ namespace OpenRCT2::TrackMetaData
-        { TrackGroup::slopeCurveLargeBanked,               TrackPitch::Up25,   TrackPitch::Up25,   TrackRoll::Right,       TrackRoll::Right,     16 }, // TrackElemType::RightEighthBankToDiagUp25
-        { TrackGroup::slopeCurveLargeBanked,               TrackPitch::Down25, TrackPitch::Down25, TrackRoll::Left,        TrackRoll::Left,     -16 }, // TrackElemType::LeftEighthBankToDiagDown25
-        { TrackGroup::slopeCurveLargeBanked,               TrackPitch::Down25, TrackPitch::Down25, TrackRoll::Right,       TrackRoll::Right,    -16 }, // TrackElemType::RightEighthBankToDiagDown25
-        { TrackGroup::slopeCurveLargeBanked,               TrackPitch::Up25,   TrackPitch::Up25,   TrackRoll::Left,        TrackRoll::Left,      16 }, // TrackElemType::LeftEighthBankToOrthogonalUp25
-        { TrackGroup::slopeCurveLargeBanked,               TrackPitch::Up25,   TrackPitch::Up25,   TrackRoll::Right,       TrackRoll::Right,     16 }, // TrackElemType::RightEighthBankToOrthogonalUp25 
-        { TrackGroup::slopeCurveLargeBanked,               TrackPitch::Down25, TrackPitch::Down25, TrackRoll::Left,        TrackRoll::Left,     -16 }, // TrackElemType::LeftEighthBankToOrthogonalDown25
-        { TrackGroup::slopeCurveLargeBanked,               TrackPitch::Down25, TrackPitch::Down25, TrackRoll::Right,       TrackRoll::Right,    -16 }, // TrackElemType::RightEighthBankToOrthogonalDown25
-        { TrackGroup::diagBrakes,            TrackPitch::None,           TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0  }, // TrackElemType::DiagBrakes
-        { TrackGroup::diagBlockBrakes,      TrackPitch::None,           TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0  }, // TrackElemType::DiagBlockBrakes
+        // TYPE                                          VANGLE END                    VANGLE START                BANK END                BANK START              PREVIEW Z OFFSET
+        { TrackGroup::straight,                          TrackPitch::None,             TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0   }, // TrackElemType::Flat                                         
+        { TrackGroup::stationEnd,                        TrackPitch::None,             TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0   }, // TrackElemType::EndStation                                   
+        { TrackGroup::stationEnd,                        TrackPitch::None,             TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0   }, // TrackElemType::BeginStation                                 
+        { TrackGroup::stationEnd,                        TrackPitch::None,             TrackPitch::None,           TrackRoll::None,        TrackRoll::None,        0   }, // TrackElemType::MiddleStation                                
```

Comment:
Not sure I followed how the first element is changing for almost every entry on this array

Reply:
The information was incorrect before (initialised to 0), but until this PR it was almost never used. The construction window essentially hardcodes this information.